frontend/public/components/nav/section.tsx

```diff
@@ -21,6 +21,42 @@ const navSectionStateToProps = (state: RootState, {required}) => {
   };
 };
 
+const mapChild = (c, activeChild, flags, activeNamespace) => {
+  if (!c) {
+    return null;
+  }
+  const { name, required, disallowed } = c.props;
+  const requiredArray = required ? _.castArray(required) : [];
+  const requirementMissing = _.some(requiredArray, flag => (
+    flag && (flagPending(flags.get(flag)) || !flags.get(flag))
+  ));
+  if (requirementMissing) {
+    return null;
+  }
+  if (disallowed && (flagPending(flags.get(disallowed)) || flags.get(disallowed))) {
+    return null;
+  }
+  return React.cloneElement(c, {
+    key: name,
+    isActive: name === activeChild,
+    activeNamespace,
+    flags,
+  });
+};
+
+const mergeElements = (Children, PluginChildren) => {
+  PluginChildren.forEach( pc => {
+    if (pc && pc.props.mergeAfter) {
```

Comment:
Nit-pick: the `pc` part of condition shouldn't be necessary (?)

Reply:
formerly the `pc` could be null - the `_.compact()` was farther modified by `map()`.
But reworked due to change above.